app/models/event/question.rb

```diff
@@ -24,18 +24,35 @@
 
 class Event::Question < ActiveRecord::Base
   include Globalized
+  include I18nEnums
+
+  # ensure all translated attributes including subclasses are added here
+  # as globalize will add it to the base class' translated_attribute_names
+  # anyway and break sti subclasses
   translates :question, :choices
```

Comment:
Im Bezug auf https://github.com/hitobito/hitobito/pull/2769#discussion_r1732781477 @kronn : 

Ich habe es versucht, es ist aus meiner Sicht aber nicht wirklich möglich, Globalize und STI zu vereinbaren. Ich denke es ist ok nur die eine, übergeordnete `Event::Question::Translation`-Klasse zu haben, sodass die Translations auch nur in der `event_questions_translations`-Tabelle gespeichert werden. Die zusätzlichen Attribute (hier z.B. `choice` werden ja auch in der Parent-Tabelle `event_questions` gespeichert). 

Der Kommentar soll helfen, da in Zukunft nicht zu vergessen